package.json

```
@@ -16,6 +16,6 @@
   "dependencies": {
     "@ons/design-system": "^72.10.9",
```

```suggestion
    "@ons/design-system": "72.10.9",
```
If we are going to pin the dependancies, we should do it for all. Theres no saying that this couldn't be compromised too